### ISSUE TITLE
Sixel graphics not displayed

### ISSUE THREAD

**Issue by agvxov** [tmux/tmux#3745]
### Sixel graphics are not displayed
Trying to output a sixel image will result in blank space seemingly equal to the image's intended display size.

This is what i would expect, working outside of tmux:
![sixel_-_without_tmux](https://github.com/tmux/tmux/assets/114135762/ee180aad-ca69-4c60-83f5-e21dfd762640)
This is how the same looks from inside tmux:
![sixel_-_with_tmux](https://github.com/tmux/tmux/assets/114135762/607c087f-8a0d-4d87-91eb-efd3fba3f27a)

### Required information

* tmux version (`tmux -V`).
```
tmux 3.3a
```

* Platform (`uname -spa`).
```
Linux anon872387634598 6.5.7-gentoo #1 SMP Fri Oct 13 23:11:26 CEST 2023 x86_64 Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz GenuineIntel GNU/Linux
```

* $TERM inside and outside of tmux (`echo $TERM`).
```
xterm-256color # outside
tmux-256color  # inside
```

The shown tmux instance was compiled with `./configure --enable-sixel && make`.
I'm able to reproduce this with a git upstream version (381c00a) of tmux,
with an empty `.tmux.conf`.

* Logs from tmux (`tmux kill-server; tmux -vv new`).
[tmux-client-3674.log](https://github.com/tmux/tmux/files/13294942/tmux-client-3674.log)
[tmux-out-3676.log](https://github.com/tmux/tmux/files/13294943/tmux-out-3676.log)
[tmux-server-3676.log](https://github.com/tmux/tmux/files/13294944/tmux-server-3676.log)

**Comment by nicm** [comment]
3.3 doesn't support SIXEL.

**Comment by agvxov** [comment]
@nicm
then what does, why is --enable-sixel recognized and what am i misinterpreting regarding this: "https://github.com/tmux/tmux/commit/dfbc6b1888c110cf0ade66f20188c57757ee1298"?

**Comment by nicm** [comment]
That is way later than 3.3a, you need to build tmux from master.

**Comment by agvxov** [comment]
Sorry, I posted the version of my system installation by accident.
It's actually `tmux next-3.4`. However, the logs prove that testing was done on the "correct" version.

**Comment by nicm** [comment]
tmux is sending your image to the terminal. Both the incoming image and the image tmux sends to the terminal show up fine for me both inside and outside tmux: 
[in.sixel.txt](https://github.com/tmux/tmux/files/13297700/in.sixel.txt)
[out.sixel.txt](https://github.com/tmux/tmux/files/13297702/out.sixel.txt)

<img width="1002" alt="Screenshot 2023-11-08 at 14 20 52" src="https://github.com/tmux/tmux/assets/304243/4adcfcf7-2b85-462c-a053-731c531bcf6e">

**Comment by nicm** [comment]
Maybe try a different terminal?

**Comment by agvxov** [comment]
Right, turns out it does works using konsole:

![Screenshot_2023-11-08_16-54-20](https://github.com/tmux/tmux/assets/114135762/59b82204-1ac5-4a8e-99c9-fc3692cc9318)

I wonder how it goes wrong xterm.

**Comment by nicm** [comment]
If you cat the `out.sixel` file I gave in xterm outside tmux, does it show?

**Comment by agvxov** [comment]
It outputs nothing outside of tmux,
but inside it leaves out the correct amount of space with new lines again.

**Comment by topcat001** [comment]
Are you launching xterm as `xterm -ti vt340`?

**Comment by agvxov** [comment]
yes I am. and as it's show above lsix does work (outside of tmux). cating *.sixel does not (either way).
![Screenshot_2023-11-19_01-35-32](https://github.com/tmux/tmux/assets/114135762/e9911f73-170c-4736-be9b-145d6419044b)

**Comment by topcat001** [comment]
Actually, it does not work for me either in xterm! I think the image might be too big in some way. Here is a small sixel image which I wrote by hand. Can you try this?
[test.sixel.txt](https://github.com/tmux/tmux/files/13401822/test.sixel.txt)

**Comment by topcat001** [comment]
Ok so you might be hitting some limit in xterm there. I usually use it  to display graphs which are small images. Also `tools/image.sixel` in the tmux git is a good test.

**Comment by ThomasDickey** [comment]
It doesn't display in xterm because your image says that it is 1798 pixels wide (which is bigger than the available area).
```
0;0;0q"1;1;1798;
```

(the small image displays okay, but it is so small that you may not notice it).

**Comment by topcat001** [comment]
I never tried oversized images (my usecase is 800x600-ish math plots). Makes sense. In fact `out.sixel.txt` reports 1800x1500!
